_docs/guides/best-practices.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Best Practices 
```

Comment:
Thanks just so I understand you're proposing a new section (different scope) regarding what's recommended on common deployment scenarios?